distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -259,9 +260,7 @@ public void testMetacardTransformersFromRest() {
                 .log()
                 .all()
                 .assertThat()
-                .body(hasXPath("/metacard[@id='" + id + "']"))
-                .body(hasXPath("/metacard/string[@name='point-of-contact']"),
-                        containsString("Guest@Guest"));
+                .body(hasXPath("/metacard[@id='" + id + "']"));
```

Comment:
I explained it in one of my previous responses, but basically these itests aren't sending user credentials which means `point-of-contact` doesn't get set since they don't have an email.